### DISCUSSION TITLE
Error when try run yarn dev after install package

### DISCUSSION THREAD

**Discussion by macerd** [richardeschloss/nuxt-highcharts#58] [Q&A] [answered]
Hi, 

I just installed the package and when I try to run it I am getting the following error:

<img width="598" alt="Screenshot 2023-01-19 at 17 01 31" src="https://user-images.githubusercontent.com/16547271/213547768-83f12d9a-dfb9-4f4f-91c6-7fbbd9c4233f.png">

Some person can help me?

Steps:

1.`yarn add nuxt-highcharts`
2.`yarn add highcharts`
3. Add `nuxt-highcharts` in modules of the `nuxt.config.js` file
4.`yarn dev`

_Dependencies:_
<img width="350" alt="Screenshot 2023-01-19 at 17 10 24" src="https://user-images.githubusercontent.com/16547271/213549311-664fcec5-42a7-452d-a823-41d12257efea.png">

**Comment by richardeschloss** [comment, marked as answer]
For that version of nuxt-highcharts, you need Nuxt3. 

For nuxt2, you need to install nuxt-highcharts@2.

**Reply by macerd** [reply]
Ok, thank you. 😄
